### ISSUE TITLE
Sortieren (Reihenfolge) von 0er HM-Kanälen in Favoriten nicht möglich

### ISSUE THREAD

**Issue by Baxxy13** [jens-maus/RaspberryMatic#1560]
### Describe the issue you are experiencing

Packe ich verschiedene normale Gerätekanäle (HM/HmIP) in einen Favoriten kann ich die Reihenfolge in der Editieransicht des Favoriten bestimmen. Das klapp problemlos und wird dann im Favorit auch korrekt angezeigt.

Fehler A:
Füge ich nun einen 0er HM Kanal dem Favoriten hinzu steht dieser ​in der Editieransicht immer ganz unten und lässt sich nicht verschieben.

Fehler  B:
Schaue ich mir nun meinen Favoriten an,  steht der  0er HM-Kanal immer an erster Stelle.  Bei mehreren hinzugefügten  0er HM-Kanälen steht der erste an Erster Stelle und der zweite irgendwo mittendrin.

### Describe the behavior you expected

Auch 0er HM-Kanäle sollten sich innerhalb eines Favoriten sortieren lassen

### Steps to reproduce the issue

1. Favorit anlegen und mit ein paar "Standard Kanälen" befüllen und nach Wunsch sortieren.
2. Irgendeinen 0er HM-Kanal hinzufügen
3. versuchen in der Editieransicht den 0er HM-Kanal zu "verschieben"
4. Favorit über die Startseite oder "Status und Bedienung > Favoriten" betrachten
5. der 0er HM-Kanal steht ist immer in der "ersten Zeile" zu sehen

Favoriten Editier-Ansicht:
![RM_HM_CH0_FavEdit](https://user-images.githubusercontent.com/54898249/143281033-a253e3ce-f08f-4319-84df-004d882e4fe8.JPG)

Gespeicherter Favorit:
![RM_HM_CH0_FavView](https://user-images.githubusercontent.com/54898249/143281129-83708305-a467-4458-8336-8a6f146137c3.JPG)

### What is the version this bug report is based on?

3.61.5.20211124 - Nightly

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
- keine Logfile Einträge vorhanden
- keine Fehlermeldungen in der Browserkonsole
```

### Additional information

0er IP-Kanäle lassen sich problemlos verschieben und werden dann an der erwarteten Position angezeigt

**Comment by jens-maus** [comment]
Dieses Problem wird sicherlich den gleichen Ursprung wie das Problem haben, das auch unter "Status und Bedienung -> Geräte" die :0 Kanäle immer ganz unten einsortiert werden und sich auch die Sortierungsreihenfolge nicht wirklich ändert wenn man auf den Tabellenkopf drückt.

**Comment by Baxxy13** [comment]
Ja, könnte zusammenhängen. Aber irgendwo muss das ja herkommen.
Ich glaube eQ-3 wollte wirklich nicht das wir die 0er HM-Kanäle "zur freien Verfügung" stellen.

**Comment by jp112sdl** [comment]
> Ich glaube eQ-3 wollte wirklich nicht das wir die 0er HM-Kanäle "zur freien Verfügung" stellen.

Oder andersrum - hätten sie es gewollt, hätten sie sich nicht bemüht, die Kanäle auszublenden. ;-)
[Aber unsere Meinungen zum Sinn oder Unsinn des Kanal 0 hatten wir ja hinreichend diskutiert.](https://github.com/jens-maus/RaspberryMatic/discussions/1482#discussioncomment-1563654)

Ich vertrete nach wie vor den Standpunkt, den Kanal 0 nur für die Geräteeinstellungen einzublenden, um Raum/Gewerk zuordnen zu können.
Alles andere bleibt eine weitere neu aufgemachte Baustelle.

**Comment by Baxxy13** [comment]
Ich hätte jetzt nicht erwartet das sich ein Favorit großartig anders verhält wie ein Raum oder Gewerk. Von der eigenen Sortiermöglichkeit mal abgesehen.
Da wir die Möglichkeit aber nun haben sollte sie auch funktionieren.

Das ganze wird noch verrückter.
Meine "Fehler B" Beschreibung oben ist nicht korrekt. Der 0er HM steht nicht immer ganz oben. Er erscheint immer in der Zeile wie er (in der Reihenfolge) hinzugefügt wurde. 
0er als erstes hinzufügen= Zeile 1
0er als zweites hinzufügen = Zeile 2 
usw.

Interessant sind auch CUxD 0er.
Die stehen zwar in der Editier-Ansicht auch unten, lassen sich aber verschieben. Sieht man aber während des Editierens nicht.
In der Favoriten Ansicht wechseln sie aber korrekt ihre Position.

Sehr verwirrend.
